### DISCUSSION TITLE
谷歌系浏览器, 如何自定义悬浮翻译指定段落的快捷键, 目前似乎并无相应设置选项？

### DISCUSSION THREAD

**Discussion by XMB-7** [immersive-translate/immersive-translate#1238] [Help]
如题, 在edge的插件快捷键设置页面, 只有翻译整个页面的槽位, 而没有翻译当前悬浮段落的槽位.
![image](https://github.com/immersive-translate/immersive-translate/assets/65625632/642db09e-c2c6-4966-8452-d31f25e66770)

---

以上无论把哪个设置成alt+i, 都基本是整页翻译, 而并非插件页面上所显示的翻译该段. 这或许是一个bug? 还是我哪里没理解对.
![image](https://github.com/immersive-translate/immersive-translate/assets/65625632/fd9f4d97-7892-48de-929b-c7f1bce7d46d)

**Comment by theowenyoung** [comment]
鼠标悬停 目前只支持 Ctrl, Shift, Alt, CMD 作为同时触发的快捷键。

因为只有这些快捷键有按住的状态。其他快捷键属于一次性的。

不过似乎也可以考虑支持

**Reply by XMB-7** [reply]
希望能够支持一下, 感谢. 
我之所以想用alt+i, 是因为我在windows下有使用ahk写自动化脚本, 然后偶尔会出现脚本执行完毕某一功能键保持被按下的状态(具体底层为啥这样不太了解, 就在ahk论坛里获悉似乎有这么个状况), 所以彼时就会直接误触发鼠标悬浮翻译段落, 在我完全不需要的时候.
